scenario/deploy.xml

```diff
@@ -85,6 +87,7 @@
                 <!-- Config update end -->
                 <item name="set-admin-url" xsi:type="object" priority="1000">Magento\MagentoCloud\Step\Deploy\InstallUpdate\Update\SetAdminUrl</item>
                 <item name="setup" xsi:type="object" priority="1100">Magento\MagentoCloud\Step\Deploy\InstallUpdate\Update\Setup</item>
+                <item name="split-db" xsi:type="object" priority="1100">Magento\MagentoCloud\Step\Deploy\SplitDbConnection</item>
```

Comment:
The same priority is in the previous step, please use a different value.

Reply:
Done